.env

```diff
@@ -0,0 +1 @@
+FLASK_APP=manage.py
```

Comment:
Let's add a comment here on why this .env file is required and mention all other variables other than FLASK _APP needs to be set in tasking-manager.env file.